docs/src/dataDisplay/table/index.jsx

```diff
@@ -1,210 +1,443 @@
 
-                {/* Definition Table */}
-                <TableSampleDefinition />
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Single Line Table
+                    </Heading>
 
-                {/* Fixed Table */}
-                <TableSampleFixed />
+                    <Typography
+                        variant="body1"
+                    >
+                        A table&rsquo;s cells will not wrap content in them.
+                    </Typography>
 
-                {/* Font Size */}
-                <TableSampleFontSize />
+                    <Typography
+                        variant="body2"
+                    >
+                        <span className="font-weight-semibold">Note:</span>
+                        &nbsp;
+                        Must have
+                        <code>fixed</code>
+                        enabled.
+                    </Typography>
+                </MarkdownContainer>
 
-                {/* Full Width Table */}
-                <TableSampleFullWidth />
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleSingleLine').default}
+                >
+                    <TableSampleSingleLine />
+                </Example>
 
-                {/* Selectable Table */}
-                <TableSampleSelectable />
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Size
+                    </Heading>
 
-                {/* Single Line Table */}
-                <TableSampleSingleLine />
+                    <Typography
+                        variant="body1"
+                    >
+                        A table&rsquo;s default cell size (vertical gutters) can be changed.
+                    </Typography>
+                </MarkdownContainer>
 
-                {/* Size */}
-                <TableSampleSize />
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleSize').default}
+                >
+                    <TableSampleSize />
+                </Example>
 
-                {/* Stretch Table */}
-                <TableSampleStretch />
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Stretch Table
+                    </Heading>
+                </MarkdownContainer>
 
-                {/* Striped Table */}
-                <TableSampleStriped />
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleStretch').default}
+                >
+                    <TableSampleStretch />
+                </Example>
 
```

Comment:
We removed a "Striped" Table example from here.  Was that intentional?

Reply:
Yes. No need for it anymore.

Reply:
Resolving.